### ISSUE TITLE
Click & hold then move mouse then release causes an infinite loop firing the event:delete event.

### ISSUE THREAD

**Issue by Havenro** [antoniandre/vue-cal-v4#437]
Hi I been using vue-cal for a while now and must start of with I LOVE IT!!!!

I have in my project enabled the option :cell-click-hold="true" and :drag-to-create-event="false".
Then in my :on-event-create="onEventCreate" i have changed the default duration to 1 hour my setting the event.end to one hour earlier and the event.endTimeMinutes - 60.

Then an event is created in the calendar but what I would like to achieve is to keep pressing the mouse button down and drag the one hour event to my desired time slot. This works fine GUI-wise (except that the newly created event is not dragged but remains in it's original location until releasing the mouse button, however this to me it cosmetics I can live with) but in the background when I release the mouse button, the event-delete event it called over and over until a "Maximum call stack size exceeded" is thrown. and also the event-drop (which is of course is expected) :)

Basically what I am after is a similar event creation feature as in an iPhone where you push and hold and then select where to drop the event since it's usually quite hard to pinpoint the exact start time :)

Am I doing something wrong or failing to do something?

**Comment by rubjo** [comment]
Hi! Please provide a minimal reproduction demonstrating the issue using CodePen, CodeSandbox or similar.

**Comment by antoniandre** [comment]
Closing due to inactivity. please reopen with repro if not solved.
